src/main/java/no/unit/nva/model/Publication.java

```diff
@@ -121,10 +121,13 @@ public void updateDoiRequestStatus(DoiRequestStatus requestedStatusChange) {
                 ERROR_MESSAGE_UPDATEDOIREQUEST_MISSING_DOIREQUEST);
         }
 
-        setDoiRequest(getDoiRequest().copy()
+        Instant now = Instant.now();
+        DoiRequest updatedDoiRequest = getDoiRequest().copy()
```

Comment:
Allowed to use the `var updateDoiRequest =`  and `var now = Instant.now();` if you want.